meshtastic/tests/test_smoke1.py

```
@@ -1,677 +1,1206 @@
+        probe_timeout = min(max(1.0, INFO_READY_POLL_INTERVAL_SECONDS), remaining)
+        return_value, output = _run(*cli_args, timeout=probe_timeout)
+        last_output = output
+        if return_value == 0 and predicate(output):
+            return output
+        remaining = deadline - time.monotonic()
+        if remaining <= 0:
+            break
+        time.sleep(min(INFO_READY_POLL_INTERVAL_SECONDS, remaining))
+    raise AssertionError(
+        f"CLI readback never reached the expected state:\n{last_output}"
+    )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)